README.md

```diff
@@ -398,6 +398,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Isha Mehta](https://github.com/isha614)
   - [Ishan Kotian](https://github.com/Ishan-Kotian)
   - [Ivo Iliev](https://github.com/Iwi4a)
+  - [Imad BG](https://github.com/imadbg01)
```

Comment:
Add your name in line 395, after `Ikra P`.
I hope that helps. Let us know if you need any more help! 🙂

Reply:
yes,I get it , Thanks